projects/m1/013-birth-date-to-astrological-sign/solutions/DomenicoTraina/birth-date-to-number-of-days.py

```diff
@@ -0,0 +1,15 @@
+user_month=input("Enter your birth month: ")
+user_day=int(input("Enter your birth day: "))
+
+if user_month=="january":
```

Comment:
I'm wondering why you didn't use any list here since you already used them in previous exercise.

They would save you to do each single check manually.